acra-censor/acra-censor_configuration_provider.go

```diff
@@ -36,24 +41,40 @@ const (
 
 // Config shows handlers configuration: queries, tables, patterns
 type Config struct {
-	Handlers []struct {
+	Version          string `yaml:"version"`
+	IgnoreParseError bool   `yaml:"ignore_parse_error"`
+	ParseErrorsLog   string `yaml:"parse_errors_log"`
+	Handlers         []struct {
 		Handler  string
 		Queries  []string
 		Tables   []string
 		Patterns []string
 		FilePath string
 	}
-	IgnoreParseError bool   `yaml:"ignore_parse_error"`
-	ParseErrorsLog   string `yaml:"parse_errors_log"`
 }
 
+// ErrUnsupportedConfigVersion acra-censor's config has version less than MinimalCensorConfigVersion
+var ErrUnsupportedConfigVersion = errors.New("acra-censor's config is outdated")
+
 // LoadConfiguration loads configuration of AcraCensor
 func (acraCensor *AcraCensor) LoadConfiguration(configuration []byte) error {
 	var censorConfiguration Config
 	err := yaml.Unmarshal(configuration, &censorConfiguration)
 	if err != nil {
 		return err
 	}
+	configVersion, err := utils.ParseVersion(censorConfiguration.Version)
+	if err != nil {
+		return err
+	}
+	censorVersion, err := utils.ParseVersion(MinimalCensorConfigVersion)
+	if err != nil {
+		return err
+	}
+	if censorVersion.Compare(configVersion) == utils.Greater {
+		// censor has version newer than config
```

Comment:
I think we should log error in human readable form, smth like

```
acraCensor.logger.
  WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCensorSetupError).
  Errorln("AcraCensor config file version is not supported: probably AcraCensor configuration (acra-censor.yaml) is outdated, check docs for deprecation warnings https://docs.cossacklabs.com/pages/documentation-acra/#acracensor-acra-s-firewall")
```

Reply:
okay, added